csharp-mode.el

```diff
@@ -91,6 +98,15 @@
   csharp '("bool" "byte" "sbyte" "char" "decimal" "double" "float" "int" "uint"
 	   "long" "ulong" "short" "ushort" "void" "object" "string" "var"))
 
+(c-lang-defconst c-other-decl-kwds
+  csharp nil)
+
+(c-lang-defconst c-type-list-kwds
+  csharp nil)
+
+(c-lang-defconst c-other-block-decl-kwds
+  csharp nil)
+
```

Comment:
Any reason you decided to move this around, instead of change them in place?

Not a biggie, but it makes the diff harder to read :D

Reply:
I think the reason was that I found them related, so it fit my mental model of them being grouped :) I just still view the codebase as evolving, but that should probably stop by now, hehe. Should I move them back?

Reply:
No biggie. Was just something which confused me at first while reading the diff. That's all cleared up now.